public/locales/en/common.json

```diff
@@ -35,6 +35,7 @@
         "upcomingClasses": "Upcoming classes",
         "waitlistedClasses": "Waitlisted classes",
         "waitlistedInfo": "An email will be sent to you when a spot opens",
+        "waitlistCancel": "Remove from Waitlist",
```

Comment:
Will the other languages for this text be added at a later time?

Reply:
yup, just adding them as stubs for now. If they want to translate it, they can simply use something like https://www.electronjs.org/apps/i18n-manager and send us the updated locales folder.